README.md

```diff
@@ -196,47 +199,13 @@ You can use the UUIDs of the GPUs set in these environment variables to verify
 that they were handed out in a way consistent with the semantics shown in the
 figure above.
 
-Likewise, looking at the `ClaimAllocations` section of the
-`NodeAllocationState` object on the worker node will show which GPUs have been
-allocated to a given `ResourceClaim` by the resource driver:
-```console
-$ kubectl describe -n dra-example-driver nas/dra-example-driver-cluster-worker
-...
-Spec:
-  ...
-  Prepared Claims:
-    132ccf41-2ec6-4751-a0e5-94f3635a679a:
-      Gpu:
-        Devices:
-          Uuid:  GPU-0159f35e-99ee-b2b5-74f1-9d18df3f22ac
-    330d73e1-b5bb-40be-bc4b-2b940f1bf34f:
-      Gpu:
-        Devices:
-          Uuid:  GPU-18db0e85-99e9-c746-8531-ffeb86328b39
-    d764a8d4-4481-4bc6-959b-27695f434953:
-      Gpu:
-        Devices:
-          Uuid:  GPU-ee3e4b55-fcda-44b8-0605-64b7a9967744
-          Uuid:  GPU-e7b42cb1-4fd8-91b2-bc77-352a0c1f5747
-          Uuid:  GPU-9ede7e32-5825-a11b-fa3d-bab6d47e0243
-          Uuid:  GPU-f11773a1-5bfb-e48b-3d98-1beb5baaf08e
-    e811664d-e487-4eb7-9ac7-678c837cbb32:
-      Gpu:
-        Devices:
-          Uuid:  GPU-657bd2e7-f5c2-a7f2-fbaa-0d1cdc32f81b
-    ef07764f-8dc0-4c4b-a99f-328711702e63:
-      Gpu:
-        Devices:
-          Uuid:  GPU-93d37703-997c-c46f-a531-755e3e0dc2ac
-```
-
 Once you have verified everything is running correctly, delete all of the
 example apps:
 ```bash
-kubectl delete --wait=false --filename=gpu-test{1,2,3,4}.yaml
+kubectl delete --wait=false --filename=demo/gpu-test{1,2,3,4}.yaml
 ```
 
-Wait for them to terminate:
+And wait for them to terminate:
 ```console
```

Comment:
What's the reason of splitting deletion into two steps (deleting without waiting and waiting for termination)?

Reply:
This is not specific to this change (it was already there), but the reason was so that one could query to see the pods terminating instead of just waiting.